tools/data-handler/src/utils/clingo-fact-builder.ts

```diff
@@ -0,0 +1,134 @@
+export type AllowedClingoType = string | number | boolean;
+
+export type NestedBuilder = (builder: ClingoFactBuilder) => ClingoFactBuilder;
+
+export type ClingoArgument =
```

Comment:
For maintenance purposes, it might be easier if this is `ClingoArgumentInterna | NestedBuilder` 
That way we need to add new values to `ClingoArgumentInternal` only.

Reply:
Done